createApiServer.js

```diff
@@ -4,17 +4,15 @@ const assert = require('assert')
 const { join, basename } = require('path')
 const merge = require('ramda/src/merge')
 const forEachObjIndexed = require('ramda/src/forEachObjIndexed')
-const feathers = require('feathers')
-const httpLogger = require('pino-http')
+const feathers = require('@feathersjs/feathers')
+const httpLogger = require('express-pino-logger')
```

Comment:
`express-pino-logger` is literally a nothing wrapper around `pino-http`, so i reckon we should stick with `pino-http`.